### DISCUSSION TITLE
Container Box

### DISCUSSION THREAD

**Discussion by magnusbogo** [digdir/designsystemet#1874] [Components]
### Komponentbeskrivelse

Våre interne systemer bruker containers i dag for å gruppere flere komponenter og tekstfelter innenfor en side. Vi har sett på om det er mulig å bruke containeren Card for å løse dette, men ser at det ikke egner seg for å vise mye informasjon samt bruke flere cards på en side. Vil det være nyttig å ha en egen Container Box komponent for dette i designsystemet med best practices? Et eksempel på hvordan vi har brukt det i dag: 
![Eksempel](https://github.com/digdir/designsystemet/assets/168070481/70611866-8495-4761-b88a-efa2009b5bad)

Definisjon på Container Box jeg fant på nett: 
A container is a defined area or UI element that holds and contains other design elements or content within a layout or composition. It acts as a visual boundary or enclosure for the content it contains. Boxes are rectangular containers that hold and organize content within a user interface. They help group related items together and provide a visual hierachy to the layout of an interface.

### Eksempel på komponenten

https://app.uxcel.com/glossary/container
https://design.mindsphere.io/patterns/container.html
https://mui.com/material-ui/react-box/

### Hvilken virksomhet eller team kommer behovet fra?

Enova SF

### Ønsker du/dere å bidra med design og/eller utvikling av komponenten?

Kanskje

**Comment by Barsnes** [comment]
Hei, tenker du også at denne burde ha `Header` og `Footer`, slik som `Card`?
Er det andre behov? 
For meg ser dette ut som ein div med bakgrunnsfarge.

Ser for meg at her kan det være mange måtar å komponere dette på, noko som også gjer at det er vanskelig å få til god tilgjengeligheit på tvers. Kan også være at det ikkje er noko nytte for tilgjengeligheit her.. 🤔 

Me har i dag `Box`, som er ein komponent for å raskt bruke bakgrunnsfarge til `neutral` fargen, endre border radius/color og legge på skygge. https://next.storybook.designsystemet.no/?path=/docs/primitives-box--docs&globals=theme:Light
